v0/destinations/mailjet/config.js

```diff
@@ -0,0 +1,23 @@
+const { getMappingConfig } = require("../../util");
+
+const CONFIG_CATEGORIES = {
+  CREATE_OR_UPDATE_CONTACT: {
+    name: "MailJetIdentifyConfig",
+    type: "identify",
+    endpoint:
+      "https://api.mailjet.com/v3/REST/contactslist/list_ID/managemanycontacts"
+  }
+};
+const MAX_BATCH_SIZE = 5000;
+
+const ACTIONS = ["addforce", "addnoforce", "remove", "unsub"];
+
```

Comment:
What is the difference between "remove" and "unsub"?

Reply:
@shrouti1507 , in remove contact will be removed from prospect list, while in unsub contact will not remove from prospect list, but email will not sent to that perticular contact